master/inline-examples/TaxonConcept-synonym.md

```diff
@@ -0,0 +1,15 @@
+```turtle
+[] a :TaxonConcept ;
+    dcterms:title "Hebe imbricata sec. Bayly & Kellow (2006)" ;
+    :taxonName [ <https://www.ipni.org/n/989261-1> a :TaxonName ;
+            :taxonNameString "Hebe imbricata" ;
+            dwc:scientificNameAuthorship "Cockayne & Allen" ;
+            dwc:namePublishedIn "Trans. & Proc. New Zealand Inst. lvii. 42 (1927) (1927)" ] ;
+    :synonym [ <https://www.ipni.org/n/812507-1> a :TaxonName ;
+            :taxonNameString "Veronica poppelwellii" ;
+            dwc:scientificNameAuthorship "Cockayne" ; 
+            dwc:namePublishedIn "Trans. & Proc. New Zealand Inst. 1915, xlviii. 200 (1916)" ] ;
+    :accordingTo [ <urn:isbn:978-0-909010-12-6> a bibo:Book ;
+            dcterms:bibliographicCitation "Bayly, M.; Kellow, A. (2006). An illustrated guide to 
```

Comment:
```suggestion
            dcterms:bibliographicCitation """Bayly, M.; Kellow, A. (2006). An illustrated guide to 
```
Multiple line strings need triple quotes in Turtle. There are a lot of places these need to be added here.